package.json

```diff
@@ -162,7 +161,7 @@
     "@ethersproject/constants": "^5.4.0",
     "@ethersproject/contracts": "^5.4.1",
     "@ethersproject/hash": "^5.4.0",
-    "@ethersproject/providers": "^5.4.5",
+    "@ethersproject/providers": "5.4.0",
```

Comment:
version drop intended?

Reply:
did you talk to SOR authors about this version drop? might be better for them to bump than for you to drop.
Alternatively, they should be using `@ethersproject` and not `ethers`, and using `^` (at-least) versioning, so that the projects can coalesce on one version.

Reply:
yeah we need this version to match the version from SOR package for types to be compatiable